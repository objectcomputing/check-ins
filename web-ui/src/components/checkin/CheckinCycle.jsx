import React from "react";
import cycle from "./checkin-cycle.png";

const CheckinCycle = ({ style = {height: "35vw", width: "35vw"} }) => (
  <div>
    <img
      style={style}
      alt="The Check-in Cycle including Expectations, Feedback, and Development"
      src={cycle}
    />
  </div>
);

export default CheckinCycle;
