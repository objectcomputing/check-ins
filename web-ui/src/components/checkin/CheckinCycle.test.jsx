import React from "react";
import CheckinCycle from "./CheckinCycle";

const style = { backgroundColor: "red" };
it("renders correctly", () => {
  snapshot(<CheckinCycle style={style} />);
});
