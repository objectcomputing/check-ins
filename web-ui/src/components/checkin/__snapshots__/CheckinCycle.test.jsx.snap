// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<div>
  <div>
    <img
      alt="The Check-in Cycle including Expectations, Feedback, and Development"
      src="/src/components/checkin/checkin-cycle.png"
      style="background-color: red;"
    />
  </div>
</div>
`;
